web/src/lib/components/photos-page/actions/asset-job-actions.svelte

```diff
@@ -32,8 +32,13 @@
   };
 </script>
 
-{#each jobs as job}
+{#each jobs as job, index}
   {#if isAllVideos || job !== AssetJobName.TranscodeVideo}
-    <MenuOption text={getAssetJobName(job)} icon={getAssetJobIcon(job)} on:click={() => handleRunJob(job)} />
+    <MenuOption
+      text={getAssetJobName(job)}
+      icon={getAssetJobIcon(job)}
+      topBorder={index === 0}
```

Comment:
I think a dedicated <MenuDivider> child component would be better than a class like this.